src/graph.rs

```diff
@@ -283,6 +283,36 @@ impl Graph {
             status.into_result()
         }
     }
+
+    /// Import the graph serialized in `graph_def`.
+    ///
+    /// `num_return_outputs` must be the number of return outputs added (i.e. the
```

Comment:
The documentation seems to be suitable for the C API, but it's somewhat misleading in the Rust API, since `num_return_outputs` or `return_outputs` don't exist as arguments. I suppose they can just be discarded here?